custom_integrations/composite/icon.txt

```diff
@@ -0,0 +1 @@
+mdi:devices
```

Comment:
This will result in an Home Assistant style integration/helper icon, which we don't allow. It might cause a user to think this integration was provided by the Home Assistant project.

../Frenck

Reply:
Ok, sorry, didn't know that. I was just trying to use the minimum possible, and I'm not an artist by any stretch of the imagination. So, I guess that also means I can't download and use the .png version of that same mdi icon.